src/data/lex.rs

```diff
@@ -404,15 +404,15 @@ impl std::fmt::Display for LiteralToken {
             Int(i) => write!(f, "{}", i),
             UnsignedInt(u) => write!(f, "{}", u),
             Float(n) => write!(f, "{}", n),
-            Str(rcstr) => {
-                let joined = rcstr
-                    .iter()
-                    .map(RcStr::as_str)
-                    .collect::<Vec<_>>()
+            Str(s) => {
+                let joined = s
+                    // .iter()
+                    // .map(Substr::as_str)
+                    // .collect::<Vec<_>>()
                     .join(" ");
                 write!(f, "{}", joined)
             }
```

Comment:
```suggestion
            Str(s) => write!(f, "{}", s.join(" ")),
```

Reply:
Thanks, forgot to clean this up and wasnt sure it would compile.